canonical-web-and-design/ubuntu.com
# Update grid patterns on index, /about, /blog, /community, /contact-us and /core pages

## Done

- updated grid markup on the following pages (and their descendants) to ensure that `.col-` elements are the only immediate children of `.row` elements:
  - /about
  - /blog
  - /community
  - /contact-us
  - /core

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check layout of main content on these pages is not broken (ignoring site-wide components like navigation and footer):
  - /about
    - /about/packages
    - /about/release-cycle
  - /blog
    - /blog/topics/design (or any topic page)
    - /blog/cloud-and-server
    - /blog/desktop
    - /blog/internet-of-things
    - /blog/press-centre
    - /blog/archives
    - /blog/(any post)
  - /community
    - /community/canonical
    - /community/code-of-conduct
    - /community/debian
    - /community/diversity
    - /community/governance
    - /community/mission
  - /contact-us
    - /contact-us/thank-you
  - /containers
  - /core
  - /core/thank-you

## Issue / Card

Fixes #5645
